geometry_msgs/PoseStamped targetPoint
uint8 uavID
---
bool success